<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="FindAdapter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Adapter</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Combo_Adapter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open EtherCat</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Combo_Slaves">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo drivers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_move">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>move</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Close EtherCat</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_TargetVelocity">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="value">
    <number>5000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Velocity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Stop">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseB">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>0</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Torque</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>268</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode of Operation</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ModeOfOperation">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplySetB">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Settings</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_TargetTorque">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>250</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_move_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>move 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Stop_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop 2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
